fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -240,6 +240,65 @@ Future<Void> tssComparison(Req req,
 	return Void();
 }
 
+ACTOR template <class Req, class Resp, class Interface, class Multi, bool P>
+Future<Void> replicaComparison(Req req,
+                               Future<ErrorOr<Resp>> fSource,
+                               uint64_t srcEndpointId,
+                               Reference<MultiInterface<Multi>> ssTeam,
+                               RequestStream<Req, P> Interface::*channel) {
+	state int srcErrorCode = error_code_success;
+	state ErrorOr<Resp> src;
+
+	wait(store(src, fSource));
+
+	if (!src.isError()) {
+		Optional<LoadBalancedReply> srcLB = getLoadBalancedReply(&src.get());
+
+		if (!srcLB.present() || !srcLB.get().error.present()) {
+			// if there is more than 1 SS in the team, attempt to verify that the other SS servers have the same
+			// data
+			state std::vector<Future<ErrorOr<Resp>>> restOfTeamFutures;
+			restOfTeamFutures.reserve(ssTeam->size() - 1);
+			for (int i = 0; i < ssTeam->size(); i++) {
```

Comment:
As you can see, we are not paying attention to QueueData metrics (like, QueueData.failedUntil) while sending requests to the replicas. And, we are not updating the QueueData metrics based on the replies from the replicas. Probably we should update the metrics in order to account for the additional traffic that is being sent to the replica storage servers when replica consistency check is enabled. Please let me know what you think. Thanks!

Reply:
Agree the metrics should be updated.